src/App.js

```diff
@@ -34,6 +35,38 @@ class App extends React.Component {
     }
   }
 
+  // Don't forget to put clisent_id into .env for good practice.
+  // Authorization and token_endpoint maybe too?
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
to combat this, you can create these functions that don't use `this` outside the scope of the class and just reference them instead